### PR TITLE
Azure Infiniband Updated Ubuntu Versions

### PR DESCRIPTION
**Pull request by melodywang060**
## Problem 1 

On the guide it says to select  the `Ubuntu Server 20.04` image, but this image is outdated and not available anymore. 

### Quick Fix 

Replace with the newest version `Ubuntu Server 24.04` 

## Problem 2 

This command `wget https://developer.download.nvidia.com/compute/cuda/repos/$distribution/x86_64/cuda-keyring_1.0-1_all.deb` yields a 404 NOT FOUND error because `cuda-keyring_1.0-1_all.deb` does not exist anymore 

### Quick Fix

Replace with the latest version, `cuda-keyring_1.1-1_all.deb`

## Problem 3 

In this command, `sudo apt-get install -y automake dh-make git libcap2 libnuma-dev libtool make pkg-config udev curl librdmacm-dev rdma-core \
    libgfortran5 bison chrpath flex graphviz gfortran tk dpatch quilt swig tcl ibverbs-utils` 

the `dpatch` package does not exist anymore - is there a replacement or can we just delete this requirement? 

## Problem 4 

Users who are not too aware of running `conda init` might exit out of the shell first. 

### Quick Fix

Adding a line that demonstrates how to enable conda and restart shell will clarify potential confusions users may have.

### CODE REVIEW

File: source/cloud/azure/aks.md [hunk 1]
```diff
@@ -91,8 +91,8 @@ $ az extension add --name aks-preview
 
 `````
 
-```console
-$ az aks nodepool add \
+```bash
+    az aks nodepool add \
```
Comment (jameslamb) [inline source/cloud/azure/aks.md:95]:
```suggestion
az aks nodepool add \
```

File: source/guides/azure/infiniband.md [hunk 2]
```diff
@@ -247,7 +247,14 @@ wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforg
 bash Mambaforge-Linux-x86_64.sh
 ```
 
-Accept the default and allow conda init to run. Then start a new shell.
+Accept the default and allow conda init to run.
+
+```shell
+~/mambaforge/bin/conda init
+
```
Comment (jameslamb) [inline source/guides/azure/infiniband.md:254]:
```suggestion
```

Let's remove this empty line, please.

**Review by jameslamb — APPROVED** [review]
Thanks! Left some small suggestions, but none of them need to block merging.

I don't know why `dpatch` was in this example, but if you ran the entire example end-to-end and it succeeded without it, I'm confident it can be removed.

**Comment by melodywang060** [comment]
thanks @jameslamb !

**Review by jacobtomlinson — APPROVED** [review]
This looks great thanks @melodywang060 (and @jameslamb for review)

**Pull request merged**